Dynamic Content Display : https://github.com/cross-solution/cross-solution.de/issues/26

Hetal2425 opened the pull request:
=> Component managed
=> Dynamic content displayed on the right hand side
=> HTML tags converted to HTML view.

cbleek commented on src/pages/post-a-job.vue:
```diff
@@ -306,27 +387,23 @@ export default {
   },
   mounted () {},
   methods: {
+    myfun () {},
```
seems unnecessary. If so, please remove

cbleek commented on src/pages/post-a-job.vue:
```diff
@@ -286,7 +368,6 @@
 <script lang="javascript">
 import YSearchRegion from '../components/SearchRegion.vue'
 import YAddress from '../components/Form/Address.vue'
-import YJob from '../components/Form/Job.vue'
```
Why you've removed the component?

Hetal2425 commented:
Moved q-splitter section to Job component. So directly get the variable values there.

If we need to share the variable between the components then also we can use the emit method to share the values.I have added that code in files.

cbleek commented:
it works
